src/Language/PureScript/Interactive.hs

```diff
@@ -217,12 +226,15 @@ handleImport
   => ImportedModule
   -> m ()
 handleImport im = do
-   st <- gets (updateImportedModules (im :))
+   st <- gets (updateImportedModules (renew im))
    let m = createTemporaryModuleForImports st
    e <- liftIO . runMake $ rebuild (map snd (psciLoadedExterns st)) m
    case e of
      Left errs -> printErrors errs
      Right _  -> put st
+   where
+   renew :: ImportedModule -> [ImportedModule] -> [ImportedModule]
```

Comment:
> However, according to my understanding, duplicate imports mean nothing

No, they are useful:

```
> import Prelude ((+))
> import Prelude as P
```

This means that I want to use `+` globally but qualify everything else. If the user typed that, we should preserve both.